### PR TITLE
remove datahub

### PR DESCRIPTION
**Pull request by domdinicola**

### CODE REVIEW

File: backend/hct_mis_api/config/settings.py [hunk 1]
```diff
@@ -124,7 +124,6 @@
     "cash_assist_datahub_mis": env.db("DATABASE_URL_HUB_MIS"),
     "cash_assist_datahub_ca": env.db("DATABASE_URL_HUB_CA"),
     "cash_assist_datahub_erp": env.db("DATABASE_URL_HUB_ERP"),
-    "registration_datahub": env.db("DATABASE_URL_HUB_REGISTRATION"),
```
Comment (patryk-dabrowski) [inline backend/hct_mis_api/config/settings.py:127]: this should be removed in the next pr after removing models from registration_datahub

File: .env.example [hunk 2]
```diff
@@ -13,7 +13,6 @@ DATABASE_URL=postgis://postgres:postgres@db:5432/postgres
 DATABASE_URL_HUB_MIS=postgis://postgres:postgres@db:5432/mis_datahub
 DATABASE_URL_HUB_CA=postgis://postgres:postgres@db:5432/ca_datahub
 DATABASE_URL_HUB_ERP=postgis://postgres:postgres@db:5432/erp_datahub
-DATABASE_URL_HUB_REGISTRATION=postgis://postgres:postgres@db:5432/rdi_datahub
```
Comment (patryk-dabrowski) [inline .env.example:16]: this should be removed in the next pr after removing models from registration_datahub

File: compose.display.yml [hunk 3]
```diff
@@ -23,7 +23,6 @@ services:
       - DATABASE_URL_HUB_MIS=postgis://postgres:postgres@db:5432/mis_datahub
       - DATABASE_URL_HUB_CA=postgis://postgres:postgres@db:5432/ca_datahub
       - DATABASE_URL_HUB_ERP=postgis://postgres:postgres@db:5432/erp_datahub
-      - DATABASE_URL_HUB_REGISTRATION=postgis://postgres:postgres@db:5432/rdi_datahub
```
Comment (patryk-dabrowski) [inline compose.display.yml:26]: same here

File: compose.selenium.yml [hunk 4]
```diff
@@ -25,7 +25,6 @@ services:
       - DATABASE_URL_HUB_MIS=postgis://postgres:postgres@db_selenium:5432/mis_datahub
       - DATABASE_URL_HUB_CA=postgis://postgres:postgres@db_selenium:5432/ca_datahub
       - DATABASE_URL_HUB_ERP=postgis://postgres:postgres@db_selenium:5432/erp_datahub
-      - DATABASE_URL_HUB_REGISTRATION=postgis://postgres:postgres@db_selenium:5432/rdi_datahub
```
Comment (patryk-dabrowski) [inline compose.selenium.yml:28]: same here

File: cypress/docker-compose.ci.yml [hunk 5]
```diff
@@ -10,7 +10,6 @@ x-django-envs: &django_envs
     - DATABASE_URL_HUB_MIS=postgis://postgres:postgres@mis_datahub_db:5432/postgres
     - DATABASE_URL_HUB_CA=postgis://postgres:postgres@cash_assist_datahub_db:5432/postgres
     - DATABASE_URL_HUB_ERP=postgis://postgres:postgres@erp_datahub_db:5432/postgres
-    - DATABASE_URL_HUB_REGISTRATION=postgis://postgres:postgres@registration_datahub_db:5432/postgres
```
Comment (patryk-dabrowski) [inline cypress/docker-compose.ci.yml:13]: same here

File: deployment/docker-compose.cy.yml [hunk 6]
```diff
@@ -11,7 +11,6 @@ x-django-envs: &django_envs
     - DATABASE_URL_HUB_MIS=postgis://postgres:postgres@mis_datahub_db:5432/postgres
     - DATABASE_URL_HUB_CA=postgis://postgres:postgres@cash_assist_datahub_db:5432/postgres
     - DATABASE_URL_HUB_ERP=postgis://postgres:postgres@erp_datahub_db:5432/postgres
-    - DATABASE_URL_HUB_REGISTRATION=postgis://postgres:postgres@registration_datahub_db:5432/postgres
```
Comment (patryk-dabrowski) [inline deployment/docker-compose.cy.yml:14]: same here

File: deployment/docker-compose.selenium-night.yml [hunk 7]
```diff
@@ -17,7 +17,6 @@ services:
       - DATABASE_URL_HUB_MIS=postgis://postgres:postgres@db:5432/mis_datahub
       - DATABASE_URL_HUB_CA=postgis://postgres:postgres@db:5432/ca_datahub
       - DATABASE_URL_HUB_ERP=postgis://postgres:postgres@db:5432/erp_datahub
-      - DATABASE_URL_HUB_REGISTRATION=postgis://postgres:postgres@db:5432/rdi_datahub
```
Comment (patryk-dabrowski) [inline deployment/docker-compose.selenium-night.yml:20]: same here

File: deployment/docker-compose.selenium.yml [hunk 8]
```diff
@@ -17,7 +17,6 @@ services:
       - DATABASE_URL_HUB_MIS=postgis://postgres:postgres@db:5432/mis_datahub
       - DATABASE_URL_HUB_CA=postgis://postgres:postgres@db:5432/ca_datahub
       - DATABASE_URL_HUB_ERP=postgis://postgres:postgres@db:5432/erp_datahub
-      - DATABASE_URL_HUB_REGISTRATION=postgis://postgres:postgres@db:5432/rdi_datahub
```
Comment (patryk-dabrowski) [inline deployment/docker-compose.selenium.yml:20]: same here

File: deployment/docker-compose.tst.yml [hunk 9]
```diff
@@ -10,7 +10,6 @@ x-django-envs: &django_envs
     - DATABASE_URL_HUB_MIS=postgis://postgres:postgres@db:5432/mis_datahub
     - DATABASE_URL_HUB_CA=postgis://postgres:postgres@db:5432/ca_datahub
     - DATABASE_URL_HUB_ERP=postgis://postgres:postgres@db:5432/erp_datahub
-    - DATABASE_URL_HUB_REGISTRATION=postgis://postgres:postgres@db:5432/rdi_datahub
```
Comment (patryk-dabrowski) [inline deployment/docker-compose.tst.yml:13]: same here

File: docker-compose.cy.yml [hunk 10]
```diff
@@ -11,7 +11,6 @@ x-django-envs: &django_envs
     - DATABASE_URL_HUB_MIS=postgis://postgres:postgres@mis_datahub_db:5432/postgres
     - DATABASE_URL_HUB_CA=postgis://postgres:postgres@cash_assist_datahub_db:5432/postgres
     - DATABASE_URL_HUB_ERP=postgis://postgres:postgres@erp_datahub_db:5432/postgres
-    - DATABASE_URL_HUB_REGISTRATION=postgis://postgres:postgres@registration_datahub_db:5432/postgres
```
Comment (patryk-dabrowski) [inline docker-compose.cy.yml:14]: same here

File: backend/hct_mis_api/config/settings.py [hunk 11]
```diff
@@ -148,17 +147,12 @@
         "sslrootcert": "/code/psql-cert.crt",
         "options": "-c search_path=erp",
     }
-    DATABASES["registration_datahub"]["OPTIONS"] = {
-        "sslmode": "verify-full",
-        "sslrootcert": "/code/psql-cert.crt",
-    }
```
Comment (patryk-dabrowski) [inline backend/hct_mis_api/config/settings.py:154]: this also should be removed after applying migration 117 in the next pr

File: backend/hct_mis_api/config/settings.py [hunk 12]
```diff
@@ -148,17 +147,12 @@
...
 
 # If app is not specified here it will use default db
 DATABASE_APPS_MAPPING: Dict[str, str] = {
     "cash_assist_datahub": "cash_assist_datahub_ca",
     "mis_datahub": "cash_assist_datahub_mis",
     "erp_datahub": "cash_assist_datahub_erp",
-    "registration_datahub": "registration_datahub",
```
Comment (patryk-dabrowski) [inline backend/hct_mis_api/config/settings.py:161]: this one too, without this line all migration from registration_datahub will be applied on the default database, but should be on registration_datahub

**Comment by domdinicola** [comment]
@patryk-dabrowski thanks for the feedback

**Review by patryk-dabrowski — APPROVED** [review]

**Review by johniak — APPROVED** [review]

**Pull request merged**